docs_src/microservices/device/sdk/Ch-DeviceSDK.md

```diff
@@ -25,16 +33,17 @@ The Device Service SDK supports:
 
 Key to diagram
 
-|Colour of Box |	Description|
```

Comment:
This Device Service Workflow figure is old and not easy to understand.  I suggest to remove this section.